Dockerfile-replica-db

```diff
@@ -0,0 +1,13 @@
+FROM alpine:3.6
```

Comment:
Also, if there is a pre made one somewhere, you can extend it with [FROM](https://docs.docker.com/engine/reference/builder/#from)

Reply:
I didn't find one. Plus this is very simple, no need to depend on something else

Reply:
Then I would do `FROM debian` which is what the official docker containers use. You can use a tag if you want a specific version:
https://hub.docker.com/_/debian/

but I guess you'll also need to use `apt-get` instead of `apk` so don't worry about changing it unless you want to. :)

Reply:
I did some reading on it before I went with alpine. For what I understand, alpine images are more memory efficient and have a smaller footprint than debian counterparts. Since this is only for dev purposes and we only need openssh-client that was enough for me to decided on alpine